File: backend/src/main/java/space/davids_digital/vk_gpt_bot/orm/repository/AppCeoRepository.java
package space.davids_digital.vk_gpt_bot.orm.repository;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import space.davids_digital.vk_gpt_bot.orm.entity.AppCeoEntity;

@Repository
public interface AppCeoRepository extends CrudRepository<AppCeoEntity, Long> {
    boolean existsByUserId(long userId);
}

File: backend/src/main/java/space/davids_digital/sweetie/orm/repository/AppCeoRepository.kt
package space.davids_digital.sweetie.orm.repository

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import space.davids_digital.sweetie.orm.entity.AppCeoEntity

@Repository
interface AppCeoRepository: CrudRepository<AppCeoEntity, Long> {
    fun existsByUserId(userId: Long): Boolean
}